src/index.ts

```diff
@@ -5,6 +5,7 @@ import { applyMiddleware as applyGraphQLMiddleware } from "graphql-middleware"
 import bodyParser from "body-parser"
 import config from "./config"
 import cors from "cors"
+import { get } from "lodash"
```

Comment:
@mzikherman - this is now safe to add as it's been moved to stage 3: https://github.com/tc39/proposal-optional-chaining

Reply:
(Also this https://github.com/tc39/proposal-nullish-coalescing, which is a slightly different take)

Reply:
Ah, actually -- the TypeScript compiler isn't yet there.